docs/visualization/matplotlib_integration.rst

```diff
@@ -69,7 +69,7 @@ precise scale and format to use for the tick labels, in which case you can make
 use of the `~astropy.visualization.time_support` function. This feature needs to
 be explicitly turned on:
 
-.. doctest-requires:: matplotlib
+.. doctest-requires:: matplotlib, pytest-mpl
```

Comment:
I'm confused - why are these needed? the doctests don't use pytest-mpl?

Reply:
The doctests apparently do use `pytest-mpl`.  Here's the traceback I get without requiring `pytest-mpl`:

```
――――――――――― [doctest] docs/visualization/matplotlib_integration.rst ――――――――――――
066
067 However, in many cases, you will probably want to have more control over the
068 precise scale and format to use for the tick labels, in which case you can make
069 use of the `~astropy.visualization.time_support` function. This feature needs to
070 be explicitly turned on:
071
072 .. doctest-requires:: matplotlib
073
074     >>> from astropy.visualization import time_support
075     >>> time_support()  # doctest: +IGNORE_OUTPUT
UNEXPECTED EXCEPTION: could not import 'pytest-mpl': No module named 'pytest-mpl'
Traceback (most recent call last):

  File "/usr/local/Cellar/python/3.7.7/Frameworks/Python.framework/Versions/3.7/lib/python3.7/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest matplotlib_integration.rst[3]>", line 1, in <module>

  File "/Users/lbradley/Dropbox/softw/packages/dev/astropy/astropy/visualization/time.py", line 50, in time_support
    from astropy.visualization.wcsaxes.utils import select_step_hour, select_step_scalar

  File "/Users/lbradley/Dropbox/softw/packages/dev/astropy/astropy/visualization/wcsaxes/__init__.py", line 9, in <module>
    pytest.importorskip("pytest-mpl")

  File "/Users/lbradley/.venvs/dev/lib/python3.7/site-packages/_pytest/outcomes.py", line 193, in importorskip
    raise Skipped(reason, allow_module_level=True) from None

Skipped: could not import 'pytest-mpl': No module named 'pytest-mpl'

/Users/lbradley/Dropbox/softw/packages/dev/astropy/docs/visualization/matplotlib_integration.rst:75: UnexpectedException
```

Reply:
yes, I don't think any pytest.importskips should be in the `__init__.py` files. And as pytest is not a mandatory runtime dependency any more, I'm having some issues with the matplotlib importskip approach, too.